### PR TITLE
New Home Page

### PR DESCRIPTION
**Pull request by cnunciato**
Closes pulumi/home#541.

### CODE REVIEW

**Comment by chrsmith** [comment]
OK, so I just took a look at the live site. OMFG it is amazing. What can I do to help get this landed?

**Comment by cnunciato** [comment]
Today's all about finishing things up on the mobile front, so I think I'm good! ⛵️ Unless of course you feel like making a sweet demo video for that big gray placeholder there....

**Comment by cnunciato** [comment]
All right, @joeduffy -- it's all yours. 

I still have a bit of refactoring I'd like to do in the HTML, CSS and JS, just to clean up the code and fix any cross-browser issues, but I can fiddle with that this weekend while you work on the word usements and languaging. ✍️

**Comment by joeduffy** [comment]
Great, thanks @cnunciato :+1:

**Comment by cnunciato** [comment]
Looks like something went awry here, with the commits -- lemme have a look.

**Comment by cnunciato** [comment]
Should be good now. 🍹

**Comment by joeduffy** [comment]
Thanks @cnunciato, looking solid. I'll whittle away at some of the marketing copy tonight, but won't touch any styling or anything like that.

I really only have one bit of styling feedback about the carousel. It's minor in the grand scheme of things, but wanted to drop it here for pondering:

I do prefer the more "plain and boring" Create | Deploy | Manage headers versus the pill buttons, for what it's worth. The similarity to the unnamed other website doesn't bug me whatsoever. My main reason is I find the small text doesn't capture your attention, and the colorization makes it hard for me to know what's actually being highlighted at this moment in time. But there's also the fact that the larger fonts more spread out uses more of that whitespace which seems quite a bit to my eyes right at the fold, where we want to capture your attention to scroll. For large monitors, you can see enough of what's below them to get you to scroll, but for others, I think that bold and strong "Create | Deploy | Manage" will help to draw your attention down after the top level H1/H2.

FWIW.

**Comment by cnunciato** [comment]
> My main reason is I find the small pill button text doesn't capture your attention, and the colorization makes it hard for me to know what's actually being highlighted at this moment in time. 

It sounds like others may be missing this, too, yeah. I'll work on making this clearer and either push up here or send up another PR post-merge.

File: assets/js/carousel.js [hunk 1]
```diff
@@ -40,9 +40,15 @@
             .css("pointer-events", "auto");
 
         $(".carousel-item-label")
-            .removeClass("bg-purple-700")
+            .removeClass("border-purple-700")
```
Comment (cnunciato) [inline assets/js/carousel.js]: Still need to do some refactoring for better maintainability here and in a few other places. Will do that post-merge unless objections.

**Pull request merged**